xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/notification/email/macros.vm

```diff
@@ -274,7 +280,7 @@
     #set ($sortedEventList = $element.value)
     #if ($wiki != '')
       #set ($wikiDescriptor = $services.wiki.getById($wiki))
-      <li style="margin: 10px 0;"><img src="cid:NEWiki.png" alt="•" style="width: 10px;"/> $escapetool.xml($wikiDescriptor.prettyName)
+      <li style="margin: 10px 0;"><img src="cid:NEWiki.png" alt="" style="width: 10px;"/> $escapetool.xml($wikiDescriptor.prettyName)
```

Comment:
Interesting... this image is actually only available in MailTemplate page, making this macro buggy for any other usage...

Reply:
The proper way to do it IMO would be to use a `gear` :gear: emoji. This is easy and light to include in the mail content. Moreover we won't really have any maintenance work and it should work whatever the page using this macro.

Reply:
Sounds like an interesting option indeed, we'd need a ticket and a new PR for that probably.

Reply:
I created https://jira.xwiki.org/browse/XWIKI-22219 :+1: